### ISSUE TITLE
Stable epsilon or specific range

### ISSUE THREAD

**Issue by yiorgosl** [Unity-Technologies/ml-agents#4984]
I have noticed from the results that epsilon decreases constantly, till it reaches around 0,1.
I need for a test to be stable to 0.6 or to go down to no more than 0.5.
Is there a way i can achieve it?
Can we have a way to control the drop rate or the limit the epsilon goes down?
Thank you.

**Comment by andrewcoh** [comment]
Hi @yiorgosl 

Epsilon is controlled by the same schedule as the learning rate. So, if you set the `learning_rate_schedule` to constant, epsilon will not decay. If you need further fine grained control, you'll have to modify code in the ppo optimizer. 

Can you share your use case for this? Just for completeness, epsilon controls the size of the PPO trust region. Epsilon of 0.6 is a bit large and may lead to some instability.

**Comment by omarathon** [comment]
+1. The [Training Configuration File docs](https://github.com/Unity-Technologies/ml-agents/blob/main/docs/Training-Configuration-File.md) says epsilon decays to 0 if one enforces an epsilon decay, but actually the code caps its decay to 0.1, as in https://github.com/Unity-Technologies/ml-agents/blob/main/ml-agents/mlagents/trainers/ppo/optimizer_torch.py. This is inconsistent.
I ran into this issue when expecting epsilon to decay to 0. My use-case is a tricky environment where there is a perfect sequence of actions to yield a +1 reward, and if any action diverges from the optimal in that sequence, then the agent is punished with a -1 reward. The need for a small epsilon increases proportional to the length of that sequence, since the probability of a random action increases in a compounding manner proportional to the number of actions the agent can execute in that sequence.
It would be nice if the docs could be updated to fix the inconsistency of the epsilon decay cap which is apparently 0 in the docs but 0.1 in the code. It would also be nice if the training configuration file enabled specification of this decay cap, since one has to edit the source code to achieve that otherwise.
